docs/markdown/icons-in-md.md

```diff
@@ -0,0 +1,204 @@
+
+
+
+## When & when not to use icons within the product
+
+**When to use**
+
+- As symbolic communication for elements that do not have room for text, like the number of pins in a carousel. In this case, ensure the icon choice is easily recognizable and makes sense to international users.
+- To convey a critical meaning that cannot be communicated with words, like a downward chevron in a Button to indicate it reveals a menu.
+- To help with quick scanning by adding rhythm and hierarchy to the design.
+
+**When NOT to use**
+
+- For purposes that are decorative or for visual embellishment, such as how illustrations are typically used. Contact us if this is needed.
+- As a visual reinforcement for associated text, without adding new meaning.
+- To communicate status or health. Use [Status](https://gestalt.netlify.app/status) instead.
+- As an interactive element (e.g., utilizing hover, focus, click/tap). Use [IconButton](https://gestalt.netlify.app/iconbutton) instead.
+
+
+## Best practices
+
+**Do**
+
+- Use icons intentionally, ensuring the Icon choice is easily recognizable and makes sense in the context.
+- Pair text and icons when possible to provide better clarity.
+- Do center-align icons when they’re next to text.
+- Do match your icon color with your text color when pairing them.
+
+**Dont**
+
+- Repurpose icons. Using icons for their intended meaning supports good comprehension.
+- Don't create interactive Icons using TapArea. Use [IconButton](https://gestalt.netlify.app/iconbutton) instead.
+- Don’t baseline-align icons to the text.
+- Don’t use different colors for text and icons.
+
+**Tips for Designing with Icons**
+
+If you decide to include icons in your interface, **do your research first**! Familiarize yourself with **icons used by other competitors** and with icons commonly used **on the platforms**, as those will be most recognizable to your users.
+
+If you must design new icons, strive to make them easy to recognize and memorize by following these guidelines:
+
+- Keep the design **simple and schematic**. Reduce the amount of graphic details by focusing on the basic characteristics of the object rather than creating a highly realistic image in order to speed up recognition. (Intricate details are difficult to distinguish at smaller sizes.)
+- Use the**5-second rule**: if it takes you more than 5 seconds to think of an appropriate icon for something, it is unlikely that an icon can effectively communicate that meaning.
+- [Test the icons for recognizability](https://www.nngroup.com/articles/icon-testing/): ask people what they expect the icons to stand for.
+- **Test the icons for memorability**: ask a repeat set of users if they can remember the icon’s meaning after being told what it represented a couple weeks earlier.
+
+
+## Icon sizing
+
+These are the guidelines for icon sizes (in px):
```

Comment:
Can this be a table?

Reply:
Probably, the original doc didn't have it a table either